registry.json

```diff
@@ -14,6 +14,21 @@
  "operatorName": "Juanma Olaizola",                                
  "github": "Olaizola13"                                               
 },
+{
+ "ticker": "SMART",                                                      
+ "poolId": "c1bc5c701c149bb7e346b47b8cc631160078b2b50e8ec37b2d0a7891", 
+ "social": {                                                           
+  "facebook": "", 
+  "twitter": "smartnodeFR",
+   "website": "https://smart-node.com/",  
+   "reddit": "",  
+   "instagram": "",
+   "youtube": ""
+ },
+ "telegram": "joinchat/AAAAAFWkMIPwtU4-FCOkeA",                                            
+ "operatorName": "SMART-NODE",                                
```

Comment:
Please use a real name or leave empty!

Reply:
hello @leo42
I have made the change

Reply:
I was talking about the Operator name...

Telegram needs to be a personal telegram handle(so we can validate votes)

Reply:
You can leave the operatorName empty but you need to provide a valid private Telegram handle.

Reply:
thank you @lophre 
I think it's ok now...